notebooks/multimodal_vdj/01_preprocessing.ipynb

```diff
@@ -0,0 +1,3169 @@
+{
```

Comment:
<p>Might be worth mentioning that the "normalization" just scales this to fractions, right? Normalization can mean a looooot of things with anything single-cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>"we view the fractions of cells in each condition instead of the absolute amount of cells." from cell above. Exchanged "." to ":" that it is clear that the plot tagged is the result of this comment.</p>